.github/workflows/format.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     - name: Configure
       run: >
         cmake -S . -B build
-        -DCMAKE_CXX_CLANG_TIDY="$(which clang-tidy);--warnings-as-errors=*"
+        -DCMAKE_CXX_CLANG_TIDY="$(which clang-tidy)"
```

Comment:
We already treat warning as errors in the config so this arg is unnecessary here.